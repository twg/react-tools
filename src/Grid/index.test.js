/* eslint-env jest */

import React from 'react'
import renderer from 'react-test-renderer'
import { Grid, GridItem } from './'

describe('Grid and GridItem', () => {
  it('Grid matches snapshot', () => {
    let component = renderer.create(<Grid />)

    expect(component.toJSON()).toMatchSnapshot()
  })

  it('Grid and GridItem matches snapshot', () => {
    let component = renderer.create(
      <Grid>
        <GridItem col="1/1" />
      </Grid>
    )

    expect(component.toJSON()).toMatchSnapshot()
  })
})
