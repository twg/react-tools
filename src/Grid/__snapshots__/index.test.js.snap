// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Grid and GridItem Grid and GridItem matches snapshot 1`] = `
<div
  className="grid   "
>
  <div
    className="gridItem         "
    style={
      Object {
        "width": "100%",
      }
    }
  />
</div>
`;

exports[`Grid and GridItem Grid matches snapshot 1`] = `
<div
  className="grid   "
/>
`;
